frontend/src/components/pages/RoutePlanningPage.tsx

```diff
@@ -5,8 +5,19 @@ import { useSearchParams } from "react-router-dom";
 import { useIsPhone } from "@/components/context/PhoneContext.tsx";
 import { createOrUpdateRoute } from "@/api/map/route_store.ts";
 import { Button } from "@/components/ui/button.tsx";
+import type { Point, RouteSegment } from "@/maps/manage/structure.ts";
```

Comment:
It would be helpful if you define a new overall data structure containing Point[] AND RouteSegment[] in structure.ts, and change operations.ts, RoutePlanningPage.tsx, and MapView.tsx accordingly.